src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -59,12 +70,29 @@ private void bindListeners(ReadOnlyPerson person) {
         email.textProperty().bind(Bindings.convert(person.emailProperty()));
         person.tagProperty().addListener((observable, oldValue, newValue) -> {
             tags.getChildren().clear();
-            person.getTags().forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+            initTags(person);
         });
     }
 
+    private String getTagColor(String tagName) {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.

Reply:
Noted. Necessary changes have been made by the group.